src/WebJobs.Extensions.ApiHub/ApiHubFileTriggerAttribute.cs

```diff
@@ -9,13 +9,15 @@ namespace Microsoft.Azure.WebJobs
     public sealed class ApiHubFileTriggerAttribute : ApiHubFileAttribute
     {
         /// <summary>
-        ///  Create an instance of an attribute that points to the file on SAAS file trigger
+        /// Create an instance of an attribute that points to the file on SAAS file trigger
```

Comment:
This attribute description is hard to understand. How about "Attribute used to bind a parameter to a SAAS file"?